xml/ns-System.Drawing.xml

```diff
@@ -1,6 +1,8 @@
 <Namespace Name="System.Drawing">
   <Docs>
-    <summary>Provides access to GDI+ basic graphics functionality. More advanced functionality is provided in the <see cref="N:System.Drawing.Drawing2D" />, <see cref="N:System.Drawing.Imaging" />, and <see cref="N:System.Drawing.Text" /> namespaces.</summary>
+    <summary>Provides access to GDI+ basic graphics functionality. More advanced functionality is provided in the <see cref="N:System.Drawing.Drawing2D" />, <see cref="N:System.Drawing.Imaging" />, and <see cref="N:System.Drawing.Text" /> namespaces.
+    System.Drawing is not supported in ASP.NET, ASP.NET Core, or Windows Services. We recommend ImageSharp for these platforms. For more information, see <see href="https://github.com/SixLabors/ImageSharp">SixLabors.ImageSharp</see>.
```

Comment:
I don't think this info belongs in the summary, and it's already stated twice, on lines 9 and 14. I would suggest consolidating that info instead and providing a clear alternative (they contradict each other).

9: "The System.Drawing namespace is not recommended for new development, due to not being supported within a Windows or ASP.NET service and it is not cross-platform. ImageSharp and SkiaSharp are recommended as alternatives."

14: "Classes within the System.Drawing namespace are not supported for use within a Windows or ASP.NET service. ...For a supported alternative, see Windows Imaging Components."

Reply:
Can the warning be put above the fold?

I completely missed it when looking at https://docs.microsoft.com/en-us/dotnet/api/system.drawing?view=dotnet-plat-ext-5.0 because it's in a remark below a very long list of classes.

I agree that having two warnings on the same page for the same thing is redundant.

> and it is not cross-platform

This is no longer accurate.

> However, since then Microsoft has released System.Drawing.Common to provide access to GDI+ graphics functionality cross-platform.

https://www.hanselman.com/blog/how-do-you-use-systemdrawing-in-net-core

Reply:
> I completely missed it when looking at https://docs.microsoft.com/en-us/dotnet/api/system.drawing?view=dotnet-plat-ext-5.0 because it's in a remark below a very long list of classes.

I missed it too and apparently so did @mahamr  and his customer. Can I move it above the classes and fix the text?

Reply:
that existing statement is what led me to ask the question in my original email that triggered all this - it wasn't clear whether or not it applied to System.Drawing.Common in ASP.NET Core/5.0+ apps.